ec/ecdatasource/deploymentdatasource/schema_apm.go

```diff
@@ -23,45 +23,97 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
-func apmResourceInfoSchema() tfsdk.Attribute {
-	// TODO should we use tfsdk.ListNestedAttributes here? - see https://github.com/hashicorp/terraform-provider-hashicups-pf/blob/8f222d805d39445673e442a674168349a45bc054/hashicups/data_source_coffee.go#L22
-	return tfsdk.Attribute{
-		Computed: true,
```

Comment:
It looks like the migration guide [recommends](https://www.terraform.io/plugin/framework/migrating/attributes-blocks/blocks-computed#framework-1) using nested attributes for computed blocks.

Reply:
I switched the computed blocks to ListNestedAttributes.